CHANGELOG.md

```diff
@@ -17,6 +17,20 @@ _With the release of `v3.0.0`, we're introducing a new changelog format in an at
 
 _The old changelog can be found in the `release-2.6` branch_
 
+<<<<<<< HEAD
```

Comment:
Pedro, you didn't fix the merge errors here.  There should be no <<<<<<< and related pieces left.